test/integration/kubernetes_test.rb

```diff
@@ -0,0 +1,152 @@
+# encoding: utf-8
```

Comment:
Can we move that file to `test/integration/k8s/test.rb`?

Reply:
Sure, this was used to match with the existing Docker tests.